src/libs/actions/Report.ts

```diff
@@ -2110,6 +2110,7 @@ function leaveRoom(reportID: string, isWorkspaceMemberLeavingWorkspaceRoom = fal
         const participantAccountIDs = PersonalDetailsUtils.getAccountIDsByLogins([CONST.EMAIL.CONCIERGE]);
         const chat = ReportUtils.getChatByParticipants(participantAccountIDs);
         if (chat?.reportID) {
+            Navigation.goBack(ROUTES.HOME);
```

Comment:
can we add a comment why we're doing this? it looks confusing for someone unfamiliar with the issue